### DISCUSSION TITLE
SVTR Model Self-Attention + CTC

### DISCUSSION THREAD

**Discussion by bely66** [PaddlePaddle/PaddleOCR#8484] [General]
Hi Everyone,

I'm using Vanilla SVTR to train recognition.
And wanted to train it on sentences of varying length, but didn't know if that was possible using the current arrangement of the model which might be different than PPOCR-v3

What I know is that Self-Attention + CTC have a great performance in capturing long sentences
and I found that it has CTC as its head
<img width="184" alt="image" src="https://user-images.githubusercontent.com/29528270/204496859-ad584276-7af9-415d-a0f8-11f1694adb6f.png">

I'm thrilled to know what you think about this?

**Comment by Topdu** [comment]
Vanilla SVTR does not support variable length training, but PPOCR-v3 yes.
